pupil-spa/src/app/virtual-keyboard/virtual-keyboard.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="question-container">
+  <form id='js-question-form' action="/check/question/{{questionNum}}" method="POST" autocomplete="off">
```

Comment:
I don't think the form should have an action or a method.  For one thing, this endpoint does not exist.